src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -91,20 +91,25 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
   const { user, profile } = useUser();
   const venueId = useVenueId();
   const venue = useSelector(currentVenueSelectorData);
+  const venueParentId = venue?.parentId;
```

Comment:
I don't know why there was no `venueParentId` var, even though it is used in the useConnect

Reply:
There was seemingly one on line 98: https://github.com/sparkletown/sparkle/pull/801/files#diff-d2dbbd9743869736d76cf73512c615856022b7a3ee80ec9b63bed997c40f7e37L98

Reply:
Which was presumably removed because line 105 changed to use `venue.parentId` not `venueParentId`: https://github.com/sparkletown/sparkle/pull/801/files#diff-d2dbbd9743869736d76cf73512c615856022b7a3ee80ec9b63bed997c40f7e37R105